src/docfx/build/legacy/LegacyExtensions.cs

```diff
@@ -39,5 +39,10 @@ public static string ToLegacyOutputPath(this LegacyManifestOutputItem legacyMani
         {
             return Path.Combine(docset.Config.SiteBasePath ?? string.Empty, legacyManifestOutputItem.OutputPathRelativeToSiteBasePath);
         }
+
+        public static string ToAbsoluteOutputPath(this string relativePath, Docset docset)
```

Comment:
>relativePath [](start = 62, length = 12)

all string will has this extension

Reply:
any advice?

---
In reply to: [199389546](https://github.com/dotnet/docfx/pull/2979#discussion_r199389546) [](ancestors = 199389546)